nexus/src/authz/actor.rs

```diff
@@ -14,18 +15,15 @@ use uuid::Uuid;
 #[derive(Clone, Debug)]
 pub struct AnyActor {
     authenticated: bool,
-    actor_id: Option<Uuid>,
+    actor_info: Option<(Uuid, Uuid)>,
```

Comment:
We might want a little struct with fields `silo_id` and `actor_id` here, since it'd be very easy to mix up which UUID is which (actor vs. Silo) in this tuple.

Reply:
Yeah, I was on the fence about it.  I generally agree.  But since all uses are within a few lines of this one, I thought it'd be less clear to have another named struct just for it. 🤷

Reply:
FWIW I agree with @plotnick